SQL/0000-00-01-Permission.sql

```diff
@@ -103,6 +103,8 @@ INSERT INTO `permissions` VALUES
     (55,'publication_propose', 'Publication - Propose a project', 2),
     (56,'publication_approve', 'Publication - Approve or reject proposed publication projects', 2),
     (57, 'candidate_dob_edit', 'Edit dates of birth', 2);
```

Comment:
line 105, change semicolon to comma. " ; " => " , "
It should pass travis after changing.

Reply:
@kongtiaowang thank you!! Fixed.